pkg/apis/batch/v1alpha1/job.go

```diff
@@ -317,6 +317,14 @@ type JobStatus struct {
 	// The resources that controlled by this job, e.g. Service, ConfigMap
 	// +optional
 	ControlledResources map[string]string `json:"controlledResources,omitempty" protobuf:"bytes,11,opt,name=controlledResources"`
+
+	// The time when the job starts running
+	// +optional
+	JobStartTime metav1.Time `json:"jobStartTime,omitempty" protobuf:"bytes,12,opt,name=jobStartTime"`
```

Comment:
l think `StartTime` is better, and `metav1.Time` should be pointer.

Reply:
Agree with @shinytang6